ecaudit/src/main/java/com/ericsson/bss/cassandra/ecaudit/logger/ChronicleAuditLoggerConfig.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2019 Telefonaktiebolaget LM Ericsson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.ericsson.bss.cassandra.ecaudit.logger;
+
+import java.nio.file.InvalidPathException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Properties;
+
+import net.openhft.chronicle.queue.RollCycle;
+import net.openhft.chronicle.queue.RollCycles;
+import org.apache.cassandra.exceptions.ConfigurationException;
+
+class ChronicleAuditLoggerConfig
+{
+    private static final String CONFIG_LOG_DIR = "log_dir";
+    private static final String CONFIG_ROLL_CYCLE = "roll_cycle";
+    private static final String CONFIG_MAX_LOG_SIZE = "max_log_size";
```

Comment:
should the unit be part of the variable name?
Cassandra seems to have that in the cassandra.yaml file...
E.g.
commitlog_total_space_in_mb
batch_size_warn_threshold_in_kb

Reply:
Once again, trying to stay aligned with Cassandra 4.0